compiler/src/iree/compiler/ConstEval/test/jit_globals_vmvx.mlir

```diff
@@ -0,0 +1,265 @@
+// RUN: iree-opt --split-input-file --iree-consteval-jit-globals --iree-consteval-jit-target-device=vmvx --iree-consteval-jit-debug --verify-diagnostics %s | FileCheck %s
```

Comment:
why vmvx? you can have two RUN statements in the same file, one per backend
if we do have any different files it should just be for things that only work in one place or the other and all things expected to work everywhere (like all these) should be common

Reply:
Split the files a bit differently. Does this make sense? (Coverage isn't complete... would like a `vmvx-expected-warning` with `--verify-diagnostics` somehow...)